core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -3629,11 +3622,12 @@ void NADefaults::initCurrentDefaultsWithDefaultDefaults()
   }
 // End: Temporary workaround for SQL build regressions to pass
 
-  // Cache all the default keywords up front,
   // leaving other non-keyword token to be cached on demand.
```

Comment:
Should this comment line (line 3625 in the new file) be deleted also? It doesn't make sense by itself.